src/commands/firestore-databases-create.ts

```diff
@@ -0,0 +1,64 @@
+import { Command } from "../command";
+import * as clc from "colorette";
+import * as fsi from "../firestore/api";
+import * as types from "../firestore/api-types";
+import { logger } from "../logger";
+import { requirePermissions } from "../requirePermissions";
+import { Emulators } from "../emulator/types";
+import { warnEmulatorNotSupported } from "../emulator/commandUtils";
+import { FirestoreOptions } from "../firestore/options";
+
+export const command = new Command("firestore:databases:create <database>")
+  .description("Create a database in your Firebase project.")
+  .option("--json", "Prints raw json response of the create API call if specified")
```

Comment:
Applies for all commands: No need to explicitly declare --json as an option - it is defined as a program option for `firebase`, not on a per command basis (https://github.com/firebase/firebase-tools/blob/master/src/index.ts#L16)

To support --json, all you need to do is have your command return the JSON that should be shown - in this case, that's databaseResp, so you should be all good.